src/api/supabase/queries/limited-case-assignments.ts

```diff
@@ -0,0 +1,11 @@
+import supabase from '../createClient';
+
+export async function getAllLCAListings() {
```

Comment:
can we add documentation here? For examples, you can look at the `/queries/interest.ts` file or `/queries/LanguageSupport.ts` file